### PR TITLE
Fixes #9703: Rudder v.3.2.9 web UI \"Content of the file\" field design was degraded.

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/9703

### CODE REVIEW

File: rudder-web/src/main/webapp/style/rudder-bootstrap.css [hunk 1]
```diff
@@ -438,7 +438,12 @@
     resize: vertical;
     border-radius: 4px !important;
 }
-
+table .tw-bs textarea.form-control,.tw-bs table textarea.form-control{
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder-bootstrap.css]: maybe the second selector is enough ? 

Should we also do it on text field ?

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by ncharles** [comment]
Shouldn't it be targeted to 3.1 ?
I just compared 3.1 and 4.0, they have the same font

**Comment by VinceMacBuche** [comment]
@ncharles You are absolutely right, i changed it in Redmine

@RaphaelGauthier you can retarget this PR using "rudder-dev retarget" when you are on the good branch

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
@RaphaelGauthier It seems it does not ahev recreated a new PR ... can you do 'rudder-dev commit VME -f" ?

**Pull request closed without merging**